sync_data/data.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Maybe remove this fie in favor of sample_act_blue_data.json in the test directory?

Reply:
yeah, it was just a test file for me to make sure my posts were working. I can pull it